LoRaEngine/modules/LoRaWanNetworkSrvModule/LoraTools/Regions/RegionManager.cs

```diff
@@ -21,10 +21,12 @@ public static bool TryTranslateToRegion(LoRaRegionType value, out Region region)
             region = null;
             switch (value)
             {
+                case LoRaRegionType.EU863: //Case added for LoRa Basics Station compatibility
```

Comment:
Do we need to update the `RegionTest` as well?

Reply:
I added to the related RegionTestData the new values